_components/icon-list/icon-list.md

```diff
@@ -19,9 +19,9 @@ subnav:
 title: Icon list
 variants:
   - variant: "`.usa-icon-list--[color]`"
-    description: Change the color of all the list's icons by updating [color] to any one of the theme colors listed on the [color utilities]({{ site.baseurl }}/utilities/color) page.
+    description: Change the color of all the list's icons by updating [color] to any one of the theme colors listed on the color utilities page.
```

Comment:
URLs are not rendering properly when included in page metadata. Removed when they are not critical to understanding